pkg/storage/bloom/v1/versioned_builder.go

```diff
@@ -55,15 +55,15 @@ func NewBlockBuilderV2(opts BlockOptions, writer BlockWriter) (*V2Builder, error
 		return nil, errors.Wrap(err, "initializing blooms writer")
 	}
 
-	return &V2Builder{
+	return &V3Builder{
 		opts:   opts,
 		writer: writer,
 		index:  NewIndexBuilder(opts, index),
 		blooms: NewBloomBlockBuilder(opts, blooms),
 	}, nil
 }
 
-func (b *V2Builder) BuildFrom(itr iter.Iterator[SeriesWithBlooms]) (uint32, error) {
+func (b *V3Builder) BuildFrom(itr iter.Iterator[SeriesWithBlooms]) (uint32, error) {
```

Comment:
We have two builders: the merge builder and the regular builder. But IIUC, we only use the `BuildFrom` from the merge builder,  right? If so, we should remove this method (in a separate PR probably).

Reply:
`BuildFrom` is actually only used in tests for building "literal blooms".